### PR TITLE
Add link to spec

### PR DESCRIPTION
**Pull request by shaedrich**

### CODE REVIEW

File: reference/datetime/dateperiod/construct.xml [hunk 1]
```diff
@@ -73,7 +73,8 @@
      <term><parameter>isostr</parameter></term>
      <listitem>
       <para>
-       An ISO 8601 repeating interval specification.
+       An <link
+        xlink:href="&url.iso-8601.Repeating_intervals;">ISO 8601 repeating interval specification</link>.
```
Comment (cmb69) [inline reference/datetime/dateperiod/construct.xml:77]: These entities  are not magically defined, but need to be defined in https://github.com/php/doc-base/blob/master/entities/global.ent before they can be used. :)

Comment (shaedrich) [reply reference/datetime/dateperiod/construct.xml:77]: Thanks for that hint! :)

Added in [#29](https://github.com/php/doc-base/pull/29)

**Review by saundefined — APPROVED** [review]

**Comment by shaedrich** [comment]
@cmb69 @saundefined I don't have write access to this repository so one of you guys has to merge it instead, I guess.

**Comment by saundefined** [comment]
@shaedrich, thank you!

**Comment by shaedrich** [comment]
You're welcome!

**Pull request merged**